profiler/src/profile_max_pool3d_fwd.cpp

```diff
@@ -234,6 +238,66 @@ int profile_max_pool3d_fwd(int argc, char* argv[])
                     pad2);
     }
 #endif
+#ifdef CK_ENABLE_FP8
```

Comment:
As above those ifdefs are not needed, since if the particular data type was not compiled, then there will be just no instances added.